Packs/Binalyze/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1,5 @@
+#### Integrations
+##### Binalyze AIR
+- All acquisition profiles support added.
+- Organization_id value type changed.
```

Comment:
As long as the user selects the organization_id from the predefined list in the command, it will not affect the user, so it does not need to be mentioned.

Reply:
Done.